design/spaces.go

```diff
@@ -27,6 +27,7 @@ var genericLinksForSpace = a.Type("GenericLinksForSpace", func() {
 	a.Attribute("workitemlinktypes", d.String, "URL to list all WILTs for this space")
 	a.Attribute("collaborators", d.String, `URL to the list of the space collaborators`)
 	a.Attribute("filters", d.String, `URL to the list of available filters`)
+	a.Attribute("categories", relationGenericList, "Space can have one or many categories")
```

Comment:
Description: it should be similar to the others cause `categories` contains URL to list all categories for this space